openwisp_users/admin.py

```diff
@@ -409,8 +409,8 @@ def get_inline_instances(self, request, obj=None):
 
     def change_view(self, request, object_id, form_url='', extra_context=None):
         extra_context = extra_context or {}
-        obj = self.model.objects.get(pk=object_id)
-        if user_not_allowed_to_change_owner(request.user, obj):
+        obj = self.get_object(request, object_id)
+        if (obj is not None) and user_not_allowed_to_change_owner(request.user, obj):
```

Comment:
the rounded parentheses are redundant, can you remove them please?

Reply:
```suggestion
        if obj is not None and user_not_allowed_to_change_owner(request.user, obj):
```

Reply:
Thanks for pointing it out, I've made the change. :+1: